doc/wiki.txt

```diff
@@ -322,8 +320,8 @@ top of a local folder of plain text Markdown files.
 ==============================================================================
 CONFIGURATION                                                     *wiki-config*
 
-The following subsections presents a list of available options and events that
-can be used to customize the behaviour of wiki.vim.
+The following subsections present a list of available options and events that
+can be used to customize the behavior of wiki.vim.
```

Comment:
Re: behavior - I've aimed for UK english, so should it not be "behaviour"?

Reply:
True, you repeat `behaviour` many times later as well. My bad.

f0956a018c2d46dd5d47fb03b2a2f16fc6c122a0